src/media/js/helpers_local.js

```diff
@@ -7,6 +7,36 @@ define('helpers_local',
              urls, user_helpers, utils_local, z) {
     var filters = nunjucks.require('filters');
     var globals = nunjucks.require('globals');
+    var dateFormat;
+
+    // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleDateString
+    // toLocaleDateString() is unreliable ; on Firefox Android and FxOS, it
+    // will return the date in m/d/Y format, even if you try to pass custom
+    // locale and/or options!.
+    // To avoid having to ship a full re-implementation, we use
+    // Intl.DateTimeFormat(), which is equivalent but only present if the
+    // underlying platform has a full implementation, and fall back to Y-m-d.
+    // This gives us a pretty date format (e.g. Saturday, February 15, 2014) on
+    // desktop, and leaves something somewhat universal on phones (2014-02-15).
+    if (typeof Intl !== 'undefined' &&
+        typeof Intl.DateTimeFormat !== 'undefined') {
+        var formatting = Intl.DateTimeFormat(user_helpers.lang(), {
+            weekday: 'long', year: 'numeric', month: 'long', day: 'numeric'
```

Comment:
Yeah. The mocks used the weekday, and I kinda like how it looks with it, but I can see how it would be distracting.